src/Helper/ValidationHelper.php

```diff
@@ -176,4 +178,26 @@ protected function validateIn(array $properties, array $values)
 
         return $valid;
     }
+
+    /**
+     * Gets the value of an attribute
+     * Throws an exception if the getter doesn't exist
+     *
+     * @param $property
+     * @return mixed
+     * @throws NostoException
+     */
+    protected function getPropertyValue($property)
+    {
+        $getter = sprintf('get%s', $property);
+        if (!method_exists($this->object, $getter)) {
+            throw new NostoException(
```

Comment:
Why not get the field value directly if there is no getter method?

Reply:
The objects we are validating don't really have public properties and if they have, it's a bad pattern and the object itself must be refactored